### PR TITLE
adds localization and description to css and updates hop description regarding leading service

### PR DESCRIPTION
**Pull request by rbertoche**
# Description

# TODO

# Media

# Changelog

### CODE REVIEW

**Comment by rbertoche** [comment]
does css supervises quartermasters? prototype still says it's hop, I just assumed it since qm is not command

**Comment by DEATHB4DEFEAT** [comment]
> does css supervises quartermasters? prototype still says it's hop, I just assumed it since qm is not command 

Quartermaster is a normal service job, CSS supervises them

File: Resources/Locale/en-US/job/job-description.ftl [hunk 1]
```diff
@@ -22,7 +23,7 @@ job-description-ertleader = Lead the Emergency Response Team in dealing with thr
 job-description-ertmedic = Ensure that the crew of the station is kept alive and healthy.
 job-description-ertsecurity = Ensure that any active threats to the station are put to rest.
 job-description-guard = Keep track of prisoners and make sure they have their basic needs. 
-job-description-hop = Handle access reassignment fairly using your ID console, manage the service department, and keep Ian safe.
+job-description-hop = Handle access reassignment fairly using your ID console, keep departments enough and well staffed, and keep Ian safe.
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/job/job-description.ftl]:
```suggestion
job-description-hop = Handle access reassignment fairly using your ID console, keep departments well staffed, and keep Ian safe.
```

File: Resources/Locale/en-US/job/job-description.ftl [hunk 2]
```diff
@@ -13,6 +13,7 @@ job-description-chef = Keep the station fed with a variety of food items, butche
 job-description-chemist = Produce medicinal drugs for the doctors to use, research ethically dubious rare chemicals, and produce weapons of war when enemies of the station arrive.
 job-description-clown = Entertain the crew through elaborate slapstick routines or terrible jokes.
 job-description-cmo = Manage the resources and personnel of the medical department to keep the crew alive and disease-free.
+job-description-css = Lead the service department which includes cargo and make sure all that the crew needs the most is provided and made available when necessary.
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/job/job-description.ftl]:
```suggestion
job-description-css = Lead the service department and make sure all things the crew needs are provided and made available when necessary.
```

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

**Comment by DEATHB4DEFEAT** [comment]
Should make changelog entries

File: Resources/Locale/en-US/job/job-description.ftl (hunk 1 above)
Comment (rbertoche) [reply Resources/Locale/en-US/job/job-description.ftl]: requested change to weird phrasing

Comment (rbertoche) [reply Resources/Locale/en-US/job/job-description.ftl]: uhhh
wrong text box

**Review by DEATHB4DEFEAT — APPROVED** [review]

File: Resources/Locale/en-US/job/job-description.ftl [hunk 3]
```diff
@@ -22,7 +23,7 @@ job-description-ertleader = Lead the Emergency Response Team in dealing with thr
 job-description-ertmedic = Ensure that the crew of the station is kept alive and healthy.
 job-description-ertsecurity = Ensure that any active threats to the station are put to rest.
 job-description-guard = Keep track of prisoners and make sure they have their basic needs. 
-job-description-hop = Handle access reassignment fairly using your ID console, manage the service department, and keep Ian safe.
+job-description-hop = Handle access reassignment fairly using your ID console, keep departments well staffed, and keep Ian safe.
```
Comment (Pspritechologist) [inline Resources/Locale/en-US/job/job-description.ftl:26]:
```suggestion
job-description-hop = Handle access reassignment fairly using your ID console, communicate among the crew and Heads, and keep Ian safe.
```

**Review by Pspritechologist — CHANGES_REQUESTED** [review]
Move them to a SimpleStation14 .ftl file, should already be a folder structure for us in there. Other than that, good change, keepin' it tidy 👍

**Comment by Pspritechologist** [comment]
Taken care of by #100 and inactive.

**Comment by rbertoche** [comment]
cool. o/

**Pull request closed without merging**